src/test/kotlin/no/nav/tilleggsstonader/sak/infrastruktur/database/repository/BehandlingRepositoryFake.kt

```diff
@@ -0,0 +1,94 @@
+package no.nav.tilleggsstonader.sak.infrastruktur.database.repository
+
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.sak.behandling.domain.Behandling
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingRepository
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingResultat
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingStatus
+import no.nav.tilleggsstonader.sak.behandling.domain.EksternId
+import no.nav.tilleggsstonader.sak.behandling.domain.Saksbehandling
+import no.nav.tilleggsstonader.sak.felles.domain.BehandlingId
+import no.nav.tilleggsstonader.sak.felles.domain.FagsakId
+import no.nav.tilleggsstonader.sak.felles.domain.FagsakPersonId
+import no.nav.tilleggsstonader.sak.util.saksbehandling
+
+class BehandlingRepositoryFake :
+    DummyRepository<Behandling, BehandlingId>({ it.id }),
+    BehandlingRepository {
+    override fun findByFagsakId(fagsakId: FagsakId): List<Behandling> {
+        TODO("Not yet implemented")
+    }
+
+    override fun findByFagsakIdAndStatus(
+        fagsakId: FagsakId,
+        status: BehandlingStatus,
+    ): List<Behandling> {
+        TODO("Not yet implemented")
+    }
+
+    override fun existsByFagsakId(fagsakId: FagsakId): Boolean {
+        TODO("Not yet implemented")
+    }
+
+    override fun finnMedEksternId(eksternId: Long): Behandling? {
+        TODO("Not yet implemented")
+    }
+
+    override fun finnAktivIdent(behandlingId: BehandlingId): String {
+        TODO("Not yet implemented")
+    }
+
+    override fun finnSaksbehandling(behandlingId: BehandlingId): Saksbehandling = saksbehandling()
```

Comment:
Jeg forklarte ikke fakes noe særlig. Men tanken min var å bruke "storage" i DummyRepository for å lagre/hente faksen fra.
Dvs man må lagre de før man kan hente de, hvis ikke er det enklest å bruke en mock.

I akkurat tilfellet med BehandlingRepostory er det litt ekstre knotete med finnSaksbehandling då den egentlige er avhengig av å få inn informasjon fra Fagsak også :/

Hvis man flytter valideringskoden din til `LæremidlerBeregnYtelseSteg` så vil man egentlige ikke ha behov for `BehandlingRepository`. Men tror at vi burde lage en fake for den også, som håndterer at en behandling er koblet til ulike typer fagsaker på en eller annen måte